packages/frameworks-solid-start/src/index.ts

```diff
@@ -13,7 +13,7 @@
  * npm install @auth/core @auth/solid-start
  * ```
  *
- * We recommended to using [create-jd-app](https://github.com/OrJDev/create-jd-app)
+ * It is recommended to use [create-jd-app](https://github.com/OrJDev/create-jd-app) to bootstrap your application with SolidStart auth included.
```

Comment:
Does nextauth normally recommend these `create` apps? Seems like this could be removed?

Reply:
Yeah no you can definitely drop that sentence in this PR if you'd like.

I have a larger PR open updating the `@auth/solid-start` package where I removed it as well. The PR updates the solid package to use the same API we've updated to over the past few months in the next.js and svelte packages (https://github.com/nextauthjs/next-auth/pull/10064), but it'll take a bit to get that in.

Reply:
Sounds good, removed it 

(also thanks for the update using the new APIs!)